src/ShipAICmd.cpp

```diff
@@ -910,7 +910,18 @@ Output("Autopilot dist = %.1f, speed = %.1f, zthrust = %.2f, state = %i\n",
 	head = head*maxdecel + perpdir*sidefactor;
 
 	// face appropriate direction
-	if (m_state >= 3) m_prop->AIMatchAngVelObjSpace(vector3d(0.0));
+	if (m_state >= 3) {
+		vector3d pos;
+		if (m_target)
+			pos = -m_dBody->GetPositionRelTo(m_target);
+		else
+			pos = -m_dBody->GetPosition();
+		m_prop->AIMatchAngVelObjSpace(vector3d(0.0));
```

Comment:
Won't this just get overridden by the subsequent call too `AIFaceDirection`?

Reply:
You could but then you'd just override it the other way around.
I think that what needs to happen is that either the ship turns around and *then* calls `AIMatchAngVelObjSpace`, or it needs to call `AIMatchAngVelObjSpace`, wait for that to complete and then call `AIFaceDirection`

Reply:
Done.